src/unlimited.cpp

```diff
@@ -1503,6 +1503,85 @@ UniValue getstat(const UniValue &params, bool fHelp)
     return ret;
 }
 
+
+
+UniValue setlog(const UniValue &params, bool fHelp)
+{
+	//Uses internal log functions
+	//Logging::
+	//Don't use them in other places.
+
+	UniValue ret = UniValue("");
+	uint64_t catg=Logging::NONE;
+	int nparm = params.size();
+	bool action = false;
+
+	if(fHelp ||nparm <1 || nparm>2)
+	{
+		throw runtime_error(
+			"log \"category|all\" \"on|off\""
+			"\nTurn categories on or off\n"
+			"\nArguments:\n"
+			"1. \"category|all\" (string, required) Category or all categories\n"
+			"2. \"on\"           (string, optional) Turn a category, or all categories, on\n"
+			"2. \"off\"          (string, optional) Turn a category, or all categories, off\n"
+			"2.                (string, optional) No argument. Show a category, or all categories, state: on|off\n"
+			 +
+			HelpExampleCli("log", "\"NET\" on") +
+			HelpExampleCli("log", "\"all\" off") +
+			HelpExampleCli("log", "\"tor\" ")+
+			HelpExampleCli("log", "\"ALL\" "));
+	}
+
+	//LOGA("LOG: Before mask: 0x%llx\n", Logging::categoriesEnabled);
+
+	try
+	{
+		if(nparm>0)
+			{
+				string category = boost::algorithm::to_upper_copy(params[0].get_str());
```

Comment:
since `dev` support c++11 I'd prefer something like:
```c++
std::string udata;
std::transform(data.begin(), data.end(), std::back_inserter(udata), ::toupper);
```

Reply:
I can do this. From here on in.
(Found boost in other places so figured this is the way things are done.)

Reply:
We have no official preference WRT std or boost, however, most programmers tend to move away from boost when a feature is included into the std library because the std library is officially sanctioned.  So if the boost implementation is somehow preferable (cleaner API, better performance, better functionality) go ahead and use it, but otherwise lets use std.

However, I would merge this either way.